src/modules/mavlink/mavlink_mission.cpp

```diff
@@ -1426,6 +1426,7 @@ MavlinkMissionManager::parse_mavlink_mission_item(const mavlink_mission_item_t *
 		case MAV_CMD_NAV_VTOL_TAKEOFF:
 		case MAV_CMD_NAV_VTOL_LAND:
 			mission_item->nav_cmd = (NAV_CMD)mavlink_mission_item->command;
+			mission_item->params[1] = mavlink_mission_item->param2;  // Transition heading
```

Comment:
Isn't the transition heading only for VTOL_TAKEOFF?

Reply:
True. While VTOL_LAND has 2 additional parameters that were not stored.
Fixed.